style.css

```diff
@@ -382,3 +365,202 @@ form textarea {
   display: flex;
   justify-content: center;
 }
+
+.btn-transition:hover {
+  transition: width 2s, height 2s, background-color 2s, transform 2s;
+  background: #6070ff;
+  color: white;
+  transform: rotate(360deg);
+}
+
+@media only screen and (min-width: 922px) {
+  .toggleNav {
+    display: none;
+  }
+
+  nav ul {
+    display: flex;
+    list-style-type: none;
+  }
+
+  nav ul li {
+    margin: 10px;
+    padding: 10px;
+    font-family: Poppins, sans-serif;
+    margin-right: 70px;
+    font-size: 120%;
+  }
+
+  nav ul li a {
+    color: #344563;
+  }
+
+  .br {
+    display: none;
+  }
+
+  .container {
+    margin-left: 13vw;
+    margin-right: 32vw;
+  }
+
+  .h1 {
+    line-height: 60px;
+    letter-spacing: 0.38px;
+  }
+
+  .icons {
+    display: grid;
+    grid-template-columns: repeat(5, 20%);
+    list-style: none;
+    text-decoration: none;
+    padding-top: 20px;
+    align-items: flex-start;
+    width: 180px;
+  }
+
+  .connect {
+    font-size: 20px;
+  }
+
+  .card-w1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    height: 496px;
+    width: 1156px;
+    margin-top: 142px;
+    margin-bottom: 140px;
+    margin-left: 8vw;
```

Comment:
- [ ] Double-check your margins, something is making your design moves. Follow the original design.

**Your design:**

![image](https://user-images.githubusercontent.com/60631456/121703593-9fd04b80-ca98-11eb-9e69-a8497a3f40a4.png)

**Original design:**

![image](https://user-images.githubusercontent.com/60631456/121703776-c5f5eb80-ca98-11eb-9560-ac27d3dc982b.png)